### PR TITLE
Update french translation

### PR DESCRIPTION
**Pull request by Bryankaveen**
I'm still not done with it, but this is my first version.

### CODE REVIEW

File: src/strings.asm [hunk 1]
```diff
@@ -261,29 +261,29 @@ string_setting_indicator:
...
+	db	'Afficher statut de la batterie',0
 string_setting_show_hidden:
-	db	'Hide hidden programs',0
+	db	'Cacher les programmes caches',0
 string_setting_poweron_password:
-	db	'Change poweron password',0
+	db	'Definir mot de passe au demarrage',0
 string_setting_editor_prgm:
-	db	'Change program editor',0
+	db	'Changer l',$27,'editeur de programmes',0
```
Comment (adriweb) [inline src/strings.asm:276]: Accent:
```suggestion
	db	'Changer l',$27,$82,'diteur de programmes',0
```

File: src/strings.asm [hunk 2]
```diff
@@ -261,29 +261,29 @@ string_setting_indicator:
...
+	db	'Appuyer <> flèches pour changer page  ',$7e,'2 sur 2]',0
 string_setting_screen_brightness:
-	db	'Change screen brightness',0
+	db	'Modifier la luminosite de l'ecran',0
 string_setting_show_battery:
-	db	'Display battery status',0
+	db	'Afficher statut de la batterie',0
 string_setting_show_hidden:
-	db	'Hide hidden programs',0
+	db	'Cacher les programmes caches',0
```
Comment (adriweb) [inline src/strings.asm:272]: Accent:
```suggestion
	db	'Cacher les programmes cach',$82,'s',0
```

File: src/strings.asm [hunk 3]
```diff
@@ -261,29 +261,29 @@ string_setting_indicator:
 string_setting_list_count:
 	db	'Afficher le nombre de programmes',0
 string_setting_page1:
-	db	'Press <> arrow keys to switch pages  ',$7e,'1 of 2]',0
+	db	'Appuyer <> flèches pour changer page  ',$7e,'1 sur 2]',0
```
Comment (adriweb) [inline src/strings.asm:264]: Order+accent:
```suggestion
	db	'Appuyer fleches <> pour changer page  ',$7e,'1 sur 2]',0
```

File: src/strings.asm [hunk 4]
```diff
@@ -261,29 +261,29 @@ string_setting_indicator:
...
 string_setting_page2:
-	db	'Press <> arrow keys to switch pages  ',$7e,'2 of 2]',0
+	db	'Appuyer <> flèches pour changer page  ',$7e,'2 sur 2]',0
```
Comment (adriweb) [inline src/strings.asm:266]: Order+accent:
```suggestion
	db	'Appuyer fleches <> pour changer page  ',$7e,'2 sur 2]',0
```

File: src/strings.asm [hunk 5]
```diff
@@ -261,29 +261,29 @@ string_setting_indicator:
...
+	db	'Modifier la luminosite de l'ecran',0
 string_setting_show_battery:
-	db	'Display battery status',0
+	db	'Afficher statut de la batterie',0
 string_setting_show_hidden:
-	db	'Hide hidden programs',0
+	db	'Cacher les programmes caches',0
 string_setting_poweron_password:
-	db	'Change poweron password',0
+	db	'Definir mot de passe au demarrage',0
```
Comment (adriweb) [inline src/strings.asm:274]: Accent:
```suggestion
	db	'D',$82,'finir mot de passe au d',$82,'marrage',0
```

File: src/strings.asm [hunk 6]
```diff
@@ -261,29 +261,29 @@ string_setting_indicator:
...
 string_setting_editor_prgm:
-	db	'Change program editor',0
+	db	'Changer l',$27,'editeur de programmes',0
 string_new_password:
-	db	'Set poweron password: ',0
+	db	'Nouveau mot de passe: ',0
 string_prgm_editor_name:
-	db	'Program editor name: ',0
+	db	'Nom de l',$27,'editeur de programmes: ',0
```
Comment (adriweb) [inline src/strings.asm:280]: Accent
```suggestion
	db	'Nom de l',$27,$82,'diteur de programmes: ',0
```

**Review by adriweb — CHANGES_REQUESTED** [review]

File: src/strings.asm [hunk 7]
```diff
@@ -261,29 +261,29 @@ string_setting_indicator:
...
 	db	'Afficher le nombre de programmes',0
 string_setting_page1:
-	db	'Press <> arrow keys to switch pages  ',$7e,'1 of 2]',0
+	db	'Appuyer <> flèches pour changer page  ',$7e,'1 sur 2]',0
 string_setting_page2:
-	db	'Press <> arrow keys to switch pages  ',$7e,'2 of 2]',0
+	db	'Appuyer <> flèches pour changer page  ',$7e,'2 sur 2]',0
 string_setting_screen_brightness:
-	db	'Change screen brightness',0
+	db	'Modifier la luminosite de l'ecran',0
```
Comment (adriweb) [inline src/strings.asm:268]:
```suggestion
	db	'Modifier la luminosit',$82,' de l',$27,$82,'cran',0
```

**Review by adriweb — CHANGES_REQUESTED** [review]
quite a few changes to make.

**Comment by Bryankaveen** [comment]
Ok, how about the "è" character?

**Comment by adriweb** [comment]
Seems like it would be `$8A`.

**Comment by mateoconlechuga** [comment]
Any luck on finishing this out?

**Comment by Bryankaveen** [comment]
It is done :D
Merry Christmas btw ;)

Le ven. 24 déc. 2021 à 23:20, Matt Waltz ***@***.***> a
écrit :

**Comment by mateoconlechuga** [comment]
Great thank you! Merry Christmas to you too :)

**Pull request merged**